.github/scripts/ci-test-stdlib.sh

```diff
@@ -22,6 +22,8 @@ declare -a tests_to_skip=(
     "SparseArrays"
     # Running LinearAlgebra in a separate job
     "LinearAlgebra"
+    # This is failing for v1.9.2 on the stock Julia as well.
+    "Profile"
```

Comment:
Should we only skip the tests inside `Profile` that are actually failing?

```suggestion
    
```

That is, consider adding this to `ci-test-patching.sh` instead:

```
    # skipping tests that also fail for stock v1.9.2+RAI Julia 
    '@test process_running(p)' "$JULIA_PATH/usr/share/julia/stdlib/v1.9/Profile/test/runtests.jl"
    '@test occursin("Overhead ╎", s)' "$JULIA_PATH//usr/share/julia/stdlib/v1.9/Profile/test/runtests.jl"
```

Reply:
Sounds good. Please feel free to make edits to the PR.